packages/meditrak-server/src/routes/importEntities/assertCanImportEntities.js

```diff
@@ -6,9 +6,9 @@ import { getCountryCode } from '@tupaia/utils';
 import { TUPAIA_ADMIN_PANEL_PERMISSION_GROUP } from '../../permissions';
 
 export const assertCanImportEntities = async (accessPolicy, entitiesByCountryName) => {
```

Comment:
It seems this function requires just country names now:

```suggestion
export const assertCanImportEntities = async (accessPolicy, countryNames) => {
```

Reply:
Thorough review for a quick pR! I've made that change, good suggestion